Example/ConsentViewController_ExampleTests/Extensions/GDPRWebViewExtensionsSpec.swift

```diff
@@ -29,11 +29,16 @@ class TestWebView: WKWebView, WKNavigationDelegate {
 
 class GDPRWebViewExtensionsSpec: QuickSpec {
     override func spec() {
+        beforeSuite {
+            // changing AsyncDefaults make the test suite pass in CI due to slow CI environment
+            AsyncDefaults.timeout = .seconds(5)
```

Comment:
we might have to increase it to 20s sometimes...

Reply:
Done